src/convertToObject.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
+// const complexStylesString = require('./stylesString');
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newString = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `newString`, as it will decrease code readability. A more descriptive name like `styleDeclarations` would be clearer.